debian/rules

```diff
@@ -22,9 +22,21 @@ override_dh_auto_build:
 	sed -i -e 's|<img alt="Travis CI status" src="https://travis-ci.org/letsencrypt/lets-encrypt-preview.svg?branch=master" />|Travis CI status|' $(CURDIR)/debian/doc-build/html/intro.html
 	sed -i -e 's|<img alt="Coverage status" src="https://coveralls.io/repos/letsencrypt/lets-encrypt-preview/badge.svg?branch=master" />|Coverage status|' $(CURDIR)/debian/doc-build/html/intro.html
 	sed -i -e 's|<img alt="Documentation status" src="https://readthedocs.org/projects/letsencrypt/badge/" />|Documentation status|' $(CURDIR)/debian/doc-build/html/intro.html
+	sed -i -e 's|<img alt="Docker Repository on Quay.io" src="https://quay.io/repository/letsencrypt/lets-encrypt-preview/status" />|Docker Repository status|' $(CURDIR)/debian/doc-build/html/intro.html
 
 override_dh_auto_install:
 	dh_auto_install
 	 
 	# Don't install the tests 
```

Comment:
I don't think there's any point in shipping the tests to end-users, but I would like to run them (once they pass) at build time so that users only receive packages that pass the tests.